### DISCUSSION TITLE
About Trilium logo

### DISCUSSION THREAD

**Discussion by HellSupernova** [TriliumNext/Trilium#2778] [Q&A]
Hi, I like the old logo very much and managed to change the logo for the app. But how can I change the logo back to the original ones inside the application? like the one on the top left corner.
 I am not a programmer and dont understand most of the code so thats why  I am asking.
Sorry to bother with this little question.
![资源 1icon 下午4 43 23](https://user-images.githubusercontent.com/71308408/162371637-96e3b311-1eea-45f0-ad31-5cf2c1fc8a8b.png)

**Comment by zadam** [comment]
That would be difficult without programming knowledge. Sorry.

**Comment by XXXJetfireXXX** [comment]
You mean like a phone app or the desktop version? I know how to do that on desktop using a custom theme, but as for the first one - no idea unfortunately.
